examples/inverse/plot_mixed_source_space_inverse.py

```diff
@@ -0,0 +1,235 @@
+# in the sub structures) based on dSPM inverse solutions
+
+# Compute the fwd matrix
+fwd = make_forward_solution(fname_raw, fname_trans, src, fname_bem,
+                            fname_fwd,
+                            mindist=5.0,  # ignore sources<=5mm from innerskull
+                            meg=True, eeg=False,
+                            n_jobs=2,
+                            overwrite=True)
+
+leadfield = fwd['sol']['data']
+print("Leadfield size : %d sensors x %d dipoles" % leadfield.shape)
+
+src_fwd = fwd['src']
+n = sum(src_fwd[i]['nuse'] for i in range(len(src_fwd)))
+print('the fwd src space contains %d spaces and %d points' % (len(src_fwd), n))
+
+# Load data
+raw = Raw(fname_raw, preload=True)
+noise_cov = mne.read_cov(fname_cov)
+events = mne.read_events(fname_event)
+
+# Add a bad channel
+raw.info['bads'] += ['MEG 2443']
+
+# Pick MEG channels
+picks = mne.pick_types(raw.info, meg=True, eeg=False, stim=False, eog=True,
+                       exclude='bads')
+
+# Define epochs for left-auditory condition
+event_id, tmin, tmax = 1, -0.2, 0.5
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=picks,
+                    baseline=(None, 0), reject=dict(mag=4e-12, grad=4000e-13,
+                                                    eog=150e-6))
+
+# Compute inverse solution and for each epoch
+snr = 1.0           # use smaller SNR for raw data
+inv_method = 'MNE'  # sLORETA, MNE, dSPM
+parc = 'aparc'      # the parcellation to use, e.g., 'aparc' 'aparc.a2009s'
+
+lambda2 = 1.0 / snr ** 2
+
+# Compute inverse operator
+inverse_operator = make_inverse_operator(raw.info, fwd, noise_cov,
+                                         loose=None, depth=None,
+                                         fixed=False)
+
+stcs = apply_inverse_epochs(epochs, inverse_operator, lambda2, inv_method,
+                            pick_ori=None, return_generator=True)
+
+# Get labels for FreeSurfer 'aparc' cortical parcellation with 34 labels/hemi
+labels_parc = mne.read_labels_from_annot(sbj_id, parc=parc,
+                                         subjects_dir=sbj_dir)
+
+# Average the source estimates within each label of the cortical parcellation
+# and each sub structure contained in the src space
+# If mode = 'mean_flip' this option is used only for the surface cortical label
+src = inverse_operator['src']
+label_ts = mne.extract_label_time_course(stcs, labels_parc, src,
+                                         mode='mean_flip',
+                                         allow_empty=True,
+                                         return_generator=False)
+
+# We compute the connectivity in the alpha band and plot it using a circular
+# graph layout
+fmin = 8.
+fmax = 13.
+sfreq = raw.info['sfreq']  # the sampling frequency
+con, freqs, times, n_epochs, n_tapers = spectral_connectivity(
+    label_ts, method='pli', mode='multitaper', sfreq=sfreq, fmin=fmin,
+    fmax=fmax, faverage=True, mt_adaptive=True, n_jobs=1)
```

Comment:
it's ok for me! 
I created two files, one for source loc and the other one for connectivity